app/assets/stylesheets/application.scss

```diff
@@ -92,4 +92,17 @@ $grey10: #C6CACC;
     padding-bottom: 5px;
     padding-left: 11px;
     padding-right: 11px;
+}
+
+.itemcard{
+    border: none;
+}
+
+.itemcard .card-body img{
+    border-radius: 5px;
+}
```

Comment:
Why do you need these two classes? It seems to me like itemcard does nothing right now. Can they be combined into one?

Reply:
I want to specifically target the itemcard. My reasoning: If we later decide that we want to use a different looking card layout somewhere else, then we won't have to touch this CSS as it does not interfere with the other layout.

And for your question:
- `.itemcard` is the top-level card (custom for the item)
- `.card-body` is a bootstrap builtin class for styling the card body

The lower selector is so long that we don't accidentally target something else.